modules/menu/main/index.coffee

```diff
@@ -31,7 +31,7 @@ getAllItems = (menu) ->
 # If setActive is called on click, the dropdown is already hidden so
 # offsetParent checking doesn't work.
 setActive = (menu, pos, up, click) ->
-  menu.dropdown.dropdown?.selectAll('.hx-menu-item').classed('hx-menu-active',false)
+  menu.dropdown._.dropdown?.selectAll('.hx-menu-item').classed('hx-menu-active',false)
```

Comment:
I think `dropdown.dropdown` may be used in more places than autocomplete/menu. A quick search brought up inline-picker but there may be more